### PR TITLE
first version of the album end point

### LINKED ISSUE #68: Create the record's specific page
/record/{id} should redirect to a page containing information about a specific record, with all the info from said record + <Add to Cart> button.

### PR DESCRIPTION
**Pull request by santiagoejm**

### CODE REVIEW

**Review by glaubernespoli — CHANGES_REQUESTED** [review]
dont disable the eslint validations

File: frontend/src/Routing/Routing.jsx [hunk 1]
```diff
@@ -1,19 +1,28 @@
-import { Route, withRouter } from 'react-router-dom';
+import { Route, Switch, withRouter } from 'react-router-dom';
 import AboutUsPage from '../containers/pages/AboutUsPage';
+import AlbumEndPointPage from '../containers/pages/AlbumEndPointPage';
```
Comment (glaubernespoli) [inline frontend/src/Routing/Routing.jsx]: rename this to a proper name

File: frontend/.eslintrc [hunk 2]
```diff
@@ -28,6 +28,8 @@
     "react/react-in-jsx-scope": "off",
     "object-curly-newline": "off",
     "comma-dangle": ["error", "never"],
-    "linebreak-style": "off"
+    "linebreak-style": "off",
+    "react/destructuring-assignment": [2, "always", { "ignoreClassFields": true }],
+    "react/prop-types": "off"
```
Comment (glaubernespoli) [inline frontend/.eslintrc]: dont disable code validations

File: frontend/src/containers/AlbumEndPoint/AlbumEndPoint.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import { Button, Card, CardContent, CardMedia, Grid, Paper, Typography } from '@material-ui/core';
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import React, { useContext } from 'react';
+import MyContext from '../../context/MyContext';
+import useStyles from './Album.Styles';
+
+const AlbumEndPoint = () => {
```
Comment (glaubernespoli) [inline frontend/src/containers/AlbumEndPoint/AlbumEndPoint.jsx]: rename this to RecordItem

File: frontend/src/Routing/routes.js [hunk 4]
```diff
@@ -3,3 +3,4 @@ export const AboutUsRoute = '/aboutus';
 export const ContactUsRoute = '/contactus';
 export const LoginRoute = '/login';
 export const SignUpRoute = '/signup';
+export const AlbumEndPointRoute = '/albumendpoint';
```
Comment (glaubernespoli) [inline frontend/src/Routing/routes.js]: rename both the variable and the endpoint

RecordRoute = '/record/'

Comment (glaubernespoli) [reply frontend/src/Routing/routes.js]: you could also make it a function:

`export const RecordRoute = id => '/record/' + id;`

then you use it like

`history.push(RecordRoute(item.id));`

**Review by Manonmani-s — DISMISSED** [review]
Ok

**Review by glaubernespoli — APPROVED** [review]

**Pull request merged**